mvvm-helpers/common-converters.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Converters
+page_title: Converters 
+description: This article describe built-in converter which can be used.
+slug: common-converters
+tags: converter
+published: True
+position: 2
+---
+
+# Converters
+
+Telerik controls provides a set of different built-in __IValueConverters__ which can be used to perform conversion between different types.
```

Comment:
Isn't the UI for WPF/Silverlight suite more appropriate? Or something that refers to the particular components, rather than all Telerik controls in general?